tasks/build-test-addon.js

```diff
@@ -7,6 +7,7 @@ module.exports = function (grunt) {
   var glob = require('glob');
   var pkg = require('../package.json');
   var activeReporters = {};
+  grunt.option('firefox-bin', '/usr/bin/firefox');  // TODO more crossplatformy
```

Comment:
Agreed - this was a hack to make it run for now as for some reason the default was something weird like /usr/lib64/firefox. https://www.npmjs.com/package/fx-runner may be like what you're suggesting, but another fix may be setting the environment variable grunt-jpm pulls more intelligently. In either case I'll resolve before merging (going to wait on getting the grunt-jpm change in regardless).